compositing-2/Overview.bs

```diff
@@ -709,16 +709,15 @@ If the 'knock-out' property is disabled, each element within the group is only c
 -->
 <h3 id="pagebackdrop">The Page Group</h3>
 
-The <a href="#isolatedgroups">isolated group</a for the root element is the
+The <a href="#isolatedgroups">isolated group</a> for the root element is the
 page group. All other elements and groups are composited into this group.
 The background of the root element (if specified) is painted into the page
 group, and any filter, clip-path, mask and and opacity is then applied, before
 compositing into the <a href="#rootgroup">root group</a>.
 
 <h3 id="rootgroup">The Root Group</h3>
 The root group is the final compositing output, and is is above the page
-group. Compositing of the page group into the root group is performed with the
-specified blend mode of the root element.
+group.
 
 <p class="note">Browsers often use an infinite white, 100% opaque root group,
 for final compositing, but this is not required. One example is an iframe with a transparent background.
```

Comment:
an iframe is not a top level group so you shouldn't call that out as an example

Reply:
I think it is, though. An HTML document is one of the applications of this spec, and each frame in the frame tree of of a web page is one HTML document. The root of an iframe is a root compositing group, as I've defined it.

Each frame in a web page draws into its ancestor in a way that is not captured in this spec (I'm not sure if it is spelled out clearly in any spec?)

I have clarified this in this PR, including renaming "page group" to "root element group".

Reply:
no, this is not correct. Unless they establish stacking contexts, embedded documents don't act like groups.
Here's an SVG example: https://codepen.io/cabanier/pen/yLyLPOY
and here an HTML one: https://codepen.io/cabanier/pen/YzPzErJ

(Note that the HTML one renders incorrectly in chrome. You need to open it in Safari)

Reply:
The HTML element is a stacking context. Therefore the div in the frame in your HTML example should blend with the backdrop from the HTML element of the iframe's stacking context, which which is transparent because no background was specified on that element.

(That being said, I agree with you that maybe the HTML element shouldn't be an isolated group in some situations. This is related to the issue of backdrop roots defined for backdrop-filter, but that is out of scope for this PR.)

I agree that an `<iframe>` element is not an isolated group though.

Reply:
> The HTML element is a stacking context. Therefore the div in the frame in your HTML example should blend with the backdrop from the HTML element of the iframe's stacking context, which which is transparent because no background was specified on that element.

You are right. Stacking starts over in an iframe (otherwise things would go very wrong 
:-) ) We should file a bug against WebKit
However SVG does not start a new stacking context.

Since stacking contexts are defined to create isolated groups, I think you should remove the part about embedded documents creating root groups. Each page should have only 1 root group.

> (That being said, I agree with you that maybe the HTML element shouldn't be an isolated group in some situations. This is related to the issue of backdrop roots defined for backdrop-filter, but that is out of scope for this PR.)

We initially tried to avoid creating isolated groups but at the time, the implementors were opposed to making them behave like authors expected because of the impact to performance and the code effort.
The 'isolation' property has a proposed property 'non-isolated' that allows an author to work around group isolation. Maybe we can re-introduce it?

Reply:
> Since stacking contexts are defined to create isolated groups, I think you should remove the part about embedded documents creating root groups. Each page should have only 1 root group.

Why though? The part about filter etc operating on the output of the page group is important. Is your suggestion that an iframe's root group will always be transparent anyway, so what's the point of it?

I think it's cleaner to leave it and have the spec be defined in terms of documents, not pages.

Reply:
> Why though? 

To have a consistent rule. Why call out special behavior that isn't needed?

> The part about filter etc operating on the output of the page group is important. 

I don't understand why that makes a difference. Filters create a stack contexts...

> Is your suggestion that an iframe's root group will always be transparent anyway, so what's the point of it?

Yes, partly :-)

Reply:
Done, made the root group just a thing for the root element of the page.